### ISSUE TITLE
Migrate Travis to GitHub Actionss

### ISSUE THREAD

**Issue by leoloso** [GraphQLAPI/graphql-api-for-wp#67]
# Feature Request

Migrate the CI from Travis to GitHub Actions. There are 2 reasons for this:

- Travis will soon stop being free for open source projects
- The monorepo (#58) may contain different projects, each with its own configuration, and this cannot be easily done through a single fixed  `.travis.yml` at the root of the project

**Comment by szepeviktor** [comment]
Travis CI is a mature project.

1. You may contact them and get free Credits ASAP
2. It is easy too import configs https://docs.travis-ci.com/user/build-config-imports/#import-sources

See viktor's credits:

![kép](https://user-images.githubusercontent.com/952007/102302136-a43c0180-3f58-11eb-82f1-eb54407fe1dd.png)

**Comment by leoloso** [comment]
This is how the switch to GitHub Actions can help with the scoping of the plugin: https://github.com/symplify/package-scoper#3-generate-github-action-workflow

**Comment by szepeviktor** [comment]
A new course is starting today: "How to move to Travis CI and be happy?"
Only €1000/hour :)

**Comment by leoloso** [comment]
No, I'm not happy. I wish them very well, they gave me (and so many OS projects) a lot of good stuff, for free. I'm just explaining why I think the move is needed for my project... It's what it is.
